src/main/java/io/javalin/core/PathMatcher.kt

```diff
@@ -73,7 +73,7 @@ data class HandlerEntry(val type: HandlerType, val path: String, val handler: Ha
 
     fun matches(requestUri: String) = parser.matches(requestUri)
 
-    fun extractParams(requestUri: String): Map<String, String> = parser.extractParams(requestUri)
+    fun extractPathParams(requestUri: String): Map<String, String> = parser.extractParams(requestUri)
```

Comment:
Hopefully `parser.extractParams` -> `parser.extractPathParams` is the last one.

Reply:
They are not, I grepped the code, PathParser should be fixes as well.

paramNames -> pathParamNames

Give me a second

Reply:
im not sure about this:

    // Use param wildcard as a capturing group
    private val paramRegex = matchRegex.pattern.replace("[^/]+?", "([^/]+?)").toRegex()

Reply:
Sure about it in what sense?

Reply:
I meant that that I'm not sure if I should change it to 
```
// Use path param wildcard as a capturing group
private val pathParamRegex = matchRegex.pattern.replace("[^/]+?", "([^/]+?)").toRegex()
```
but looks like it would seem reasonable.

Reply:
Yes, looks good!

Reply:
Ok, I believe these are all of the usages. Done, I hope :)